### PR TITLE
Misc. updates

### PR DESCRIPTION
**Pull request by jglick**
@reviewbybees

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ By default, if no selector parameter is provided, the `runSelector` step selects
 upstream job.
 
 ```groovy
-def runWrapper = runSelector job: 'upstream-project-name'
+def runWrapper = runSelector 'upstream-project-name'
```
Comment (jglick) [inline README.md:50]: A single `@DataBoundConstructor` (i.e., mandatory) parameter need not be specified.

File: README.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ selector: [$class: 'SpecificRunSelector', buildNumber: UPSTREAM_BUILD_NUMBER]
 You may have an upstream job that triggers a specific downstream job by using the `build` step:
 
 ```groovy
-build ('downstream-project-name')
+build 'downstream-project-name'
```
Comment (jglick) [inline README.md:88]: Gratuitous paren.

File: README.md [hunk 3]
```diff
@@ -94,3 +94,5 @@ A possible solution to select the run that triggered your downstream job is by u
 def runWrapper = runSelector job: 'upstream-project-name', 
 selector: [$class: 'TriggeringRunSelector'] 
 ```
+
+Of course you could instead (and more explicitly) have the upstream build pass `currentBuild.number` as a build parameter.
```
Comment (jglick) [inline README.md:98]: (my recommendation)

**Comment by ghost** [comment]
This pull request originates from a [CloudBees](https://www.cloudbees.com/) employee. At CloudBees, we require that all pull requests be reviewed by other CloudBees employees before we seek to have the change accepted. If you want to learn more about our process please see [this explanation](https://github.com/reviewbybees/about#about-reviewbybees).

**Comment by alexsomai** [comment]
:+1:

**Comment by alexsomai** [comment]
Can I merge this? Or should I wait for at least one review from a CloudBees employee?

**Pull request merged**